lib/render.js

```diff
@@ -6,24 +6,25 @@ const _ = require('lodash'),
   bluebird = require('bluebird'),
   setup = require('./setup');
 
-var hbs;
+var hbs,
+  log = require('./log').setup({ file: __filename });
 
 /**
  * Create the object passed to the partial for rendering.
  * Remember, Kiln is expecting certain things in the object,
  * but it doesn't expect the FULL
  *
- * @param  {Object} data
+ * @param  {Object} state
  * @return {Object}
  */
-function composeData(data) {
+function composeData(state) {
   return  _.assign({
-    _self: data._self,
-    _layoutRef: data._layoutRef,
-    _componentSchemas: data._componentSchemas,
-    _envVars: data._envVars,
-    locals: data.locals
-  }, data._data);
+    _self: state._self,
+    _layoutRef: state._layoutRef,
+    _componentSchemas: state._componentSchemas,
+    _envVars: state._envVars,
+    locals: state.locals
+  }, state._data);
```

Comment:
Might be simplified with `_.pick`

Reply:
So I tested this and discovered why I didn't do it the first time. The properties in `_data` need to be flattened out across the returned object, everything else can be properties. I updated to include `pick` for the creation of the base object, but the `assign` should stay for now. 

I want to update this later, but it'll mean changing the base data that's passed into handlebars and could require changing the template API.